pkg/monitoring/api/prometheus/prometheus.go

```diff
@@ -0,0 +1,64 @@
+package prometheus
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	namespaceAndVMILabels = []string{"namespace", "vmi"}
+	activeSSHTunnels      = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "ssh_active_tunnels",
+			Help: "Amount of active SSH tunnels, broken down by namespace and vmi name",
+		},
+		namespaceAndVMILabels,
+	)
+	activeVNCConnections = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "vnc_active_connections",
+			Help: "Amount of active VNC connections, broken down by namespace and vmi name",
+		},
+		namespaceAndVMILabels,
+	)
+	activeConsoleConnections = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "console_active_connections",
+			Help: "Amount of active Console connections, broken down by namespace and vmi name",
```

Comment:
i think these metrics need the `kubevirt_` prefix. see our new metrics guide here, https://github.com/kubevirt/kubevirt/blob/master/docs/metrics.md